cherrypy/_cpreqbody.py

```diff
@@ -553,20 +563,6 @@ class Part(Entity):
 
     """A MIME part entity, part of a multipart entity."""
 
-    # "The default character set, which must be assumed in the absence of a
-    # charset parameter, is US-ASCII."
-    attempt_charsets = ['us-ascii', 'utf-8']
-    """A list of strings, each of which should be a known encoding.
```

Comment:
As `attempt_charsets` is now defined in `__init__`, this docstring is now lost, so the meaning of this attribute is now replaced with an explicit definition to an implicit one. As complicated as this issue is, it's important to be as explicit and descriptive as possible.